docs/activities/events/index.html

```diff
@@ -28,12 +28,13 @@ <h2>W3C WoT – Events</h2>
 		<li>
 			<strong>Upcoming</strong>
 			<ul>
-				<li><a target="_blank" href="https://events.geekle.us/nodejs/">Node.js Global Summit, May 18th, 2022</a></li>
+				
```

Comment:
maybe we should add Testfest 2022.06 reference

Reply:
I will add it to the past events since it is already too late ^^